siliconcompiler/toolscripts/ubuntu26/install-magic.sh

```
@@ -25,11 +25,27 @@ git clone $(python3 ${src_path}/_tools.py --tool magic --field git-url) magic
+
 args=
 if [ ! -z ${PREFIX} ]; then
     args=--prefix="$PREFIX"
 fi
 
-LD_FLAGS=-shared ./configure $args
+# GCC 15 (Ubuntu 26.04) defaults to C23, where `bool` is a keyword and Magic's
+# `typedef unsigned char bool;` no longer compiles. Magic's configure ignores
+# $CFLAGS from the environment, so fold -std=gnu17 into $CC to force C17 on
+# every compile.
+LD_FLAGS=-shared CC="${CC:-gcc} -std=gnu17" ./configure $args
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)